build/commands/lib/config.js

```diff
@@ -449,11 +442,7 @@ Config.prototype.buildArgs = function () {
     args[key] = getEnvConfig([key])
   }
 
-  if (this.isOfficialBuild()) {
-    args.enable_updater = true
-  }
-
-  if (!this.isBraveReleaseBuild()) {
+  if (this.isOfficialBuild() && !this.isBraveReleaseBuild()) {
     args.chrome_pgo_phase = 0
```

Comment:
this is the default for non-official-builds already
```
chrome_pgo_phase = 0
  if (!is_cronet_build && !dcheck_always_on && is_official_build &&
      # TODO(crbug.com/1336055): Update this now-outdated condition with regard
      # to chromecast and determine whether chromeos_is_browser_only is
      # obsolete.
      (is_high_end_android || is_win || is_mac || is_fuchsia ||
       (is_linux && !is_castos) ||
       (is_ios && use_blink && target_environment == "device"))) {
    chrome_pgo_phase = 2
  }
```

Reply:
`enable_dsyms = is_official_build || using_sanitizer` in build/config/apple/symbols.gni